events/planner.py

```diff
@@ -1,43 +1,211 @@
-from events.models import OptionalMeetingDates, PossibleParticipant, Event
+from functools import reduce
+from django.utils import timezone
+from datetime import datetime, timedelta
+from reminders.models import Reminder, ReminderType
+from events.models import OptionalMeetingDates, PossibleParticipant, EventParticipant
+from main.utilities import send_mail_notification, time_format, create_notification
 
 
 class EventPlanner():
 
-    def run(self, event):
-        """ run the planner in an event.
-            Find the best possible date for the meeting,
-            then excute the choice and change the DB rows accordingly.
-        """
-        meetings_possibilities = self.find_meeting(event)
+    def __init__(self, event_id):
+        self.event_id = event_id
+        self.chosen_meeting_date = None
+        self.participants_names_who_can_meet = ""
+        self.participants_names_who_cant_meet = ""
+        self.event_participants_who_can_meet = []      # type of event participant
```

Comment:
the name of the variable should be self-explanatory so maybe you can change the name and remove the comment like:
```suggestion
        self.event_participants_type_who_can_meet = []
```

Reply:
I don't think it's necessary. I wrote those comments for myself because I got confused with the types of the variables while I wrote the code and I forgot to delete them.